lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -399,6 +399,7 @@ def cmd_search_help
               'rank'        => 'Modules with a matching rank (Can be descriptive (ex: \'good\') or numeric with comparison operators (ex: \'gte400\'))',
               'ref'         => 'Modules with a matching ref',
               'reference'   => 'Modules with a matching reference',
+              'session_type'   => 'Modules with a matching session type (SMB, MySQL, Meterpreter, etc)',
```

Comment:
Not a blocker; `SMB, MySQL, PostgresQL` probably should be behind a feature flag in the help message here